libecole/src/reward/boundintegral.cpp

```diff
@@ -100,7 +103,10 @@ auto get_primal_bound(SCIP* scip) {
 	case SCIP_STAGE_SOLVED:
 		return SCIPgetPrimalbound(scip);
 	default:
-		return SCIPinfinity(scip);
+		if (SCIPgetObjsense(scip) == SCIP_OBJSENSE_MINIMIZE) {
+			return SCIPinfinity(scip);
+		}
+		return -SCIPinfinity(scip);
```

Comment:
A detail, but possible will save us some future trouble. To be consistent with `SCIPgetPrimalbound`, we should get instead here the objective limit, which will be SCIP's infinity if it has not been set. See [`SCIPgetObjlimit`](https://www.scipopt.org/doc/html/scip__prob_8c_source.php#l01490). This also simplifies the code a bit.

Reply:
Thanks for the suggestion, I made the change.  I don't believe there is a dual bound equivalent to this right?  I just left the dual bound function as is.

Reply:
Yes I think that's only for the primal bound unfortunately...